backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/respository/InMemoryShopRepository.java

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package com.google.gpay.virtualqueue.backendservice.respository;
+
+import java.util.HashMap;
+import java.util.UUID;
+
+import com.google.gpay.virtualqueue.backendservice.model.Shop;
+import com.google.gpay.virtualqueue.backendservice.model.Token;
+
+import org.springframework.stereotype.Repository;
+
+import lombok.Getter;
+
+@Repository
+@Getter
+public class InMemoryShopRepository implements VirtualQueueRepository {
+	private HashMap<UUID, Token> token_mappings = new HashMap<>();
```

Comment:
Nit. Change this to Map ?
i.e.
private Map<UUID, Token> token_mappings = new HashMap<>();